server/controllers/profile.js

```diff
@@ -114,4 +114,21 @@ exports.findAllProfiles = asyncHandler(async (req, res, next) => {
       }
 
     res.status(200).json({ profiles: profiles });
+})
+
+exports.profileSearch = asyncHandler(async (req, res, next) => {
+  const {search} = req.body;
+  const {times} = req.body;
+  const {to} = req.body;
+  const {from} = req.body;
+
+  //find either matching firstname or lastname
+  const profiles = await Profile.find({ $or: [ { firstName: { $regex: search, $options: "i" } }, { lastName: { $regex: search, $options: "i" } } ] } )
```

Comment:
Could we put this on separate lines to make it more readable?

Reply:
OK